stable/insights-agent/templates/opa/label-required.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.opa.enabled -}}
+apiVersion: insights.fairwinds.com/v1
+kind: CustomCheck
+metadata:
+  name: must-have-label
+spec:
+  output:
+    title: "Label is missing"
+    severity: 0.1
+    remediation: "Add the label"
+    category: "Reliability"
+  rego: |
+         package fairwinds
+         labelrequired[description] {
```

Comment:
how does one return, e.g. a `description` and a `severity`? Is that possible?

Reply:
Yeah, I suppose none of the custom checks I provided show that. We'll have to figure out if that's appropriate for just documentation or if we can contrive an example where it would be good to actually demonstrate it.

So regardless of what that variable is named. If you return an Object instead of a string, then that object can have severity, description, category, and title. I believe all of them will override whatever is in the check.